### PR TITLE
Remade whole original "initial_position"

### PR DESCRIPTION
**Pull request by ivanthcih**
Originally from Vincent's original branch, due to the amount of commits (a huge mess), was made to reset and make it more cleaner.

DESC:
Communicate the initial position to the Vision System when the match starts

### CODE REVIEW

File: src/main/java/competition/subsystems/pose/PoseSubsystem.java [hunk 1]
```diff
@@ -5,19 +5,27 @@
...
 import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.PropertyFactory;
 import xbot.common.subsystems.pose.BasePoseSubsystem;
 
 @Singleton
 public class PoseSubsystem extends BasePoseSubsystem {
 
     final DriveSubsystem drive;
+    public final DoubleProperty intPosX;
```
Comment (JohnGilb) [inline src/main/java/competition/subsystems/pose/PoseSubsystem.java]: we don't use these, so we can get rid of these three properties.

File: src/main/java/competition/subsystems/vision/commands/SendInitialPos.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package competition.subsystems.vision.commands;
+
+import competition.subsystems.pose.PoseSubsystem;
+import competition.subsystems.vision.VisionSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class SendInitialPos extends BaseCommand{
```
Comment (JohnGilb) [inline src/main/java/competition/subsystems/vision/commands/SendInitialPos.java]: `SendInitialPositionToRaspberryCommand`

File: src/main/java/competition/subsystems/vision/commands/SendInitialPos.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (JohnGilb) [inline src/main/java/competition/subsystems/vision/commands/SendInitialPos.java]: Where is your constructor?

**Review by JohnGilb — CHANGES_REQUESTED** [review]
Tests!

File: src/test/java/competition/subsystems/vision/SendInitialPositionToRaspberryCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+import competition.subsystems.pose.PoseSubsystem;
+import competition.subsystems.vision.commands.SendInitialPositionToRaspberryCommand;
+
+public class SendInitialPositionToRaspberryCommandTest extends VisionSubsystemTest{
+
+    SendInitialPositionToRaspberryCommand initial;
+    VisionSubsystem vision;
+    PoseSubsystem pose;
+
+
```
Comment (JohnGilb) [inline src/test/java/competition/subsystems/vision/SendInitialPositionToRaspberryCommandTest.java]: This whole class doesn't actually test the Command - just the Subsystem.

**Review by JohnGilb — DISMISSED** [review]

**Review by stephenjust — APPROVED** [review]

**Pull request merged**